shells/firefox/build.js

```diff
@@ -15,11 +15,15 @@ const main = async () => {
     join(__dirname, 'build')
   );
 
-  console.log('\nThe Firefox extension has been built!');
+  console.log(chalk.green('\nThe Firefox extension has been built!'));
+  console.log(chalk.green('You can test it by following the steps below:'));
   console.log(chalk.gray('\n# Go to the unpacked directory:'));
   console.log(`cd ${unpackedPath}`);
   console.log(chalk.gray('\n# And launch Firefox with the extension enabled:'));
+  console.log('web-ext run');
+  console.log(chalk.gray('\n# You can also test against specific Firefox versions:'));
   console.log('web-ext run --firefox=nightly');
+  console.log('web-ext run --firefox=/Applications/Firefox54.app/Contents/MacOS/firefox-bin');
```

Comment:
Perhaps in the Fx README if we link to the [Firefox releases](https://www.mozilla.org/en-US/firefox/channel/desktop/) and people download those then they can use `--firefox=firefox` or `--firefox=beta` or `--firefox=nightly` and it should just work.  I think `Firefox54.app` might be a custom location.  

Right now Firefox is 53, Beta is 54, Nightly is 55.  Of course next week those numbers will all bump up by one.